app/mailers/teacher_mailer.rb

```diff
@@ -28,6 +28,7 @@ def deny_email(teacher, reason)
 
   def teals_confirmation_email(teacher)
     @teacher = teacher
+    # Only send if teacher is a TEALS volunteer
     if !@teacher.status.nil? and @teacher.status.include? "I am a TEALS volunteer"
```

Comment:
Do you need to change this to reflect [https://github.com/tommywei110/BJC-Teacher-Tracker/pull/36](url) ?

Reply:
It was fixed in the other commit, resulted in a merge conflict here though so I fixed it here as well